### ISSUE TITLE
Editing a fillup displays calculated values

### ISSUE THREAD

**Issue by GoogleCodeExporter** [evancharlton/android-mileage#8]
```
What steps will reproduce the problem?
1. Set format to something non-standard.
2. Record a fillup
3. Edit the fillup

Expected behavior:
Shows the entered values

Actual behavior:
Displays the calculated values; does a double-calculation upon save.

Mileage version:
Android version:
Device:

Please use labels and text to provide additional information.

```

Original issue reported on code.google.com by `evancharlton` on 27 Nov 2010 at 6:07

**Comment by GoogleCodeExporter** [comment]
```
Fixed in Mileage 3.0
```

Original comment by `evancharlton` on 30 Nov 2010 at 12:57
- Changed state: **Verified**

**Comment by GoogleCodeExporter** [comment]
Original comment by `evancharlton` on 17 Apr 2011 at 8:21
- Changed state: **Released**
